3545/1/info_collector.rb

```diff
@@ -0,0 +1,49 @@
+require 'nokogiri'
+require 'open-uri'
+require 'json'
+require './points_calculator'
+
+class InfoCollector
+  def initialize(github_link)
+    @gem_name = github_link.split('/').last.capitalize
+    @doc = Nokogiri::HTML(URI.open(github_link))
+    @doc_for_used_by = Nokogiri::HTML(URI.open(github_link + '/network/dependents'))
+    @result_array = []
+  end
+
+  def stars
+    @doc.css('.social-count')[1]
+  end
+
+  def watched_by
+    @doc.css('.social-count')[0]
+  end
+
+  def forks
+    @doc.css('.social-count')[2]
+  end
+
+  def contributors
+    @doc.css("span[class ='num text-emphasized']")[3]
+  end
+
+  def issues
+    @doc.css('.Counter')
+  end
+
+  def used_by
+    @doc_for_used_by.css('a.btn-link.selected')
+  end
+
+  def pack_all
+    @result_array = [stars, watched_by]
```

Comment:
why do you need two lines for adding elements to array?

Reply:
If I make it in one line, rubocop is upset with the length of the line (more then 100). May I left it like it is now?